### PR TITLE
Pf hist

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/personal_data/portfolios/portfolio_manager.py [hunk 1]
```diff
@@ -282,6 +282,10 @@ def _load_simulated_portfolio_from_history(self):
                 for symbol, value in self.historical_portfolio_value_manager.historical_ending_portfolio.items()
             }
         )
+        for asset_dict in portfolio_amount_dict.values():
```
Comment (Herklos) [inline octobot_trading/personal_data/portfolios/portfolio_manager.py:285]: Should we add a test to ensure we don't break it again?

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/portfolios/portfolio_manager.py:285]: good idea, i will

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/portfolios/portfolio_manager.py:285]: done

File: tests/personal_data/portfolios/test_portfolio_manager.py [hunk 2]
```diff
@@ -120,3 +124,33 @@ async def test_refresh_simulated_trader_portfolio_from_order(backtesting_trader)
         update_portfolio_from_filled_order_mock.assert_not_called()
         portfolio_manager._refresh_simulated_trader_portfolio_from_order(order)
         update_portfolio_from_filled_order_mock.assert_called_once()
+
+
+async def test_load_simulated_portfolio_from_history(backtesting_trader):
+    config, exchange_manager, trader = backtesting_trader
```
Comment (Herklos) [inline tests/personal_data/portfolios/test_portfolio_manager.py:130]: 👍

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**